From 18888d66ca55ebefbf36dc74ab25c3958d56789b Mon Sep 17 00:00:00 2001
From: Mohammed Gamal <mgamal@redhat.com>
Date: Thu, 17 Nov 2022 18:56:20 +0100
Subject: [PATCH 2/3] tools: hv: Remove an extraneous "the"

RH-Author: Mohamed Gamal Morsy <mmorsy@redhat.com>
RH-MergeRequest: 5: Update hyperv-daemons for RHEL-9.2
RH-Bugzilla: 2139457
RH-Acked-by: Emanuele Giuseppe Esposito <eesposit@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Commit: [1/2] 53bf20c6a219a20e7a2ae3986a9b2ed43ac4faac

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2139457

commit f15f39fabed2248311607445ddfa6dba63abebb9
Author: Jason Wang <wangborong@cdjrlc.com>
Date:   Thu Aug 11 21:34:33 2022 +0800

    tools: hv: Remove an extraneous "the"

    There are two "the" in the text. Remove one.

    Signed-off-by: Jason Wang <wangborong@cdjrlc.com>
    Link: https://lore.kernel.org/r/20220811133433.10175-1-wangborong@cdjrlc.com
    Signed-off-by: Wei Liu <wei.liu@kernel.org>

Signed-off-by: Mohammed Gamal <mgamal@redhat.com>
---
 hv_kvp_daemon.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hv_kvp_daemon.c b/hv_kvp_daemon.c
index c06c94d..5f98bee 100644
--- a/hv_kvp_daemon.c
+++ b/hv_kvp_daemon.c
@@ -44,7 +44,7 @@
 
 /*
  * KVP protocol: The user mode component first registers with the
- * the kernel component. Subsequently, the kernel component requests, data
+ * kernel component. Subsequently, the kernel component requests, data
  * for the specified keys. In response to this message the user mode component
  * fills in the value corresponding to the specified key. We overload the
  * sequence field in the cn_msg header to define our KVP message types.
-- 
2.31.1

